resources/hooks/local/commit-msg

```diff
@@ -13,4 +13,4 @@ COMMIT_MSG_FILE=$1
 DIFF=$(git -c diff.mnemonicprefix=false --no-pager diff -r -p -m -M --full-index --no-color --staged | cat)
 
 # Run GrumPHP
-(cd "${HOOK_EXEC_PATH}" && printf "%s\n" "${DIFF}" | exec $(HOOK_COMMAND) "--git-user=$GIT_USER" "--git-email=$GIT_EMAIL" "$COMMIT_MSG_FILE")
+(cd "${HOOK_EXEC_PATH}" && printf "%s\n" "${DIFF}" | $(EXEC_GRUMPHP_COMMAND) $(HOOK_COMMAND) "--git-user=$GIT_USER" "--git-email=$GIT_EMAIL" "$COMMIT_MSG_FILE")
```

Comment:
There is an error if $GIT_USER contains spaces like 'firstname lastname'.

It should work with `"--git-user='$GIT_USER'"`

Reply:
strange, the argument is allready wrapped with quotes ... Will investigate it. Thanks for testing!
Does it work on docker + docker-compose with a single worded username?

Reply:
It worked with docker compose and a single worded username